Docs/Conventions/Commit_Message.md

```
@@ -0,0 +1,41 @@
+| style | `style: 코드 포맷 변경` |
+| refactor | `refactor(signup): 회원 가입 로직 개선` |
+| file | `file: 이미지 파일 추가` |
+| test | `test: 테스트 코드 추가` |
+| docs | `docs: README.md 업데이트` |
+| remove | `remove: 사용하지 않는 파일 제거` |
+| ci | `ci: 자동 배포 스크립트 변동` |
+| release | `release: 릴리즈 버전 1.0.3` |
+| chore | `chore: 설정파일 수정` |
+
+## 메세지 구조
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)